src/code/bank0.asm

```diff
@@ -6882,36 +6882,47 @@ data_36B0::
 
 LoadObject_OpenDoorTop::
     ld   a, $04                                   ; $36B2: $3E $04
-    call label_36C4                               ; $36B4: $CD $C4 $36
+    call addRoomStatus                            ; $36B4: $CD $C4 $36
     push bc                                       ; $36B7: $C5
     call label_35EE                               ; $36B8: $CD $EE $35
     ld   bc, data_37E1                            ; $36BB: $01 $E1 $37
     ld   de, data_36B0                            ; $36BE: $11 $B0 $36
     jp   Func_354B                                ; $36C1: $C3 $4B $35
 
 ; Set hRoomStatus depending on the map and room
-label_36C4::
+; a = new status to add
+; f = not used, but overridden
+addRoomStatus::
```

Comment:
Nitpick: I suggest to name it with `Update` rather than `Add`, and to capitalize it.

```suggestion
UpdateRoomStatus::
```

Reply:
Function name is changed.